github_tutorial.md

```diff
@@ -0,0 +1,235 @@
+
+17. Ik wil een repository toevoegingen
+	* Alleen de "owner" van de GitHub organisatie VNG-Realisatie kan dit werkelijk uitvoeren: Michiel Verhoef (michiel.verhoef@vng.nl)
+	* Maak bij een repository direct ook een README.md aan!
+	* Denk goed na over een naam en omschrijving van de repository, zodat men deze ook kan vinden.
+	* Vul zo snel mogelijk de README.md met belangrijke informatie:
+		* Beschrijf het "product: of resultaat van de repository (in ons geval meestal een standaard). Wat is het doel dat hiermee bereikt wordt.
+		* Beschrijf hoe mensen het product kunnen gebruiken. Bijvoorbeeld verwijzing naar functionele specificaties, technische specificaties (schema's), referentie-implementatie.
+		* Beschrijf hoe mensen aan de ontwikkeling kunnen bijdragen (contributing guide).
+	* Maak, zover mogelijk een goed begin. Mensen dragen makkelijker bij aan iets dat er al (in enig vorm) is, dan aan een lege repository.
+	* Nodig relevante belanghebbenden (potentiële gebruikers of deelnemers) uit om het "product" in de repository te gebruiken en eraan bij te dragen.
+
+## Werken vanuit eigen laptop met eigen tooling
+Soms wil je met je eigen editor documenten (dat kan ook code of schema zijn) bewerken. Bijvoorbeeld in XmlSpy, Eclipse, enz. Dan is het handiger om de bestanden van de repository op je eigen laptop te hebben staan en van daaruit te bewerken. Hieronder staat hoe je de versiebeheertool git kan gebruiken op je eigen laptop of pc, in combinatie met GitHub.
+Beschreven is het gebruik van git vanuit command prompt (CMD) in windows en Terminal op een Mac. Veel tools kennen echter zelf (al dan niet via een plug in) ook voorzieningen om een deel van deze stappen te doen.
+Je kan ook GitHub Desktop downloaden en installeren, waarmee je een aantal git/github dingen kunt doen.
+
+1. Ik wil een lokale werkplek beginnen van een repository
+	* Preconditie: je hebt git (dat is iets anders dan GitHub!) geïnstalleerd op je computer
+	* Preconditie: Om documenten te kunnen bewerken of toevoegen via een push request, moet je eerst een "fork" hebben gemaakt van de VNG Realisatie repository waar je in wilt werken.
+	* Preconditie: 	Om vanuit je lokale laptop te mogen pushen naar een VNG Realisatie repository, moet je een personal access token maken. Dit access token moet als "scope" hebben "repo".
+	Wanneer bij een git commando gevraagd wordt om inlognaam en wachtwoord, vul je niet je wachtwoord in, maar je personal access token.
+	* Open command prompt (Windows) of Terminal (Mac)
+	* Ga naar de map waarin de repository moet komen, bijv. met:
+  `cd /user/vngr/github`
+  	* Wanneer dit de eerste git repository is op je laptop/pc op deze plek:
+  	```
+  	git init
+  	git config --global user.name "username"
+	git config --global user.email voornaam.achternaam@vng.nl
+	```
+	(vervang in de code hierboven "username" door je eigen GitHub gebruikersnaam en "voornaam.achternaam@vng.nl" door je eigen e-mailadres)
+	
```

Comment:
Mij is dit niet gelukt, ook niet vanaf thuis waar het netwerk verder geen hindernis is.  De enige manier waarop ik met 2-weg authenticatie in kon loggen was via de GitHub desktop.??

Reply:
Dit probleem heb je inmiddels opgelost, toch?